src/os/process_management.cc

```diff
@@ -58,7 +58,13 @@ namespace os {
             std::string temp = std::string(prog) + " " + args;
             auto args = string_split_null_terminated(temp, ' ');
 
-            int err = execv(prog.c_str(), args.data());
+            int err;
+            if(prog.find("/") == std::string::npos) {
+                err = execv(prog.c_str(), args.data());
+            } else {
+                err = execvp(prog.c_str(), args.data());
+            }
```

Comment:
The conditions seem wrong. When `prog.find("/") == std::string::npos` is true means that the "/" character was not found, thats when the `PATH` environment variable should be used.

Reply:
Fixed